File: python/world_1/examples/4_condicoes_if_else/ex029_velocidade_carro/ex029.py
# Escrevas um programa que leia a velocidade de um carro.
# Se ele utlrapassar 80km/h, mostre a mensagem que ele foi multado.
# A multa vai custar R$ 7,00 por cada km acima do limite.

velocity = int(input("Informe a velocida em km/h: "))

if velocity <= 80:
    print('MUITO BEM! Você está no limite de velocidade!')
    print("Continuem sendo um bom motorista!")

else:
    print("VOCÊ FOI MULTADO!")
    fineValue = (velocity - 80) * 7
    print(f'Por ter excedido {velocity - 80}km acima do limite, \nVocê deverá pagar R${fineValue:.2f}.')

File: python/world_1/examples/4_condicoes_if_else/ex029_velocidade_carro/ex029.js
function calculate() {

    let inputKM = document.querySelector("#input1").value;
    let inputKmConverted = Number(inputKM);
    let divAnswer = document.querySelector(".answer");

    if (inputKM == "" || inputKM < 0 || isNaN(inputKmConverted)) {

        divAnswer.innerHTML = `<div class="answer error">Informe um número válido</div>`;

    } else if (inputKmConverted <= 80) {

        divAnswer.innerHTML = `<div class="answer true">Continue assim! Você está no limite de velocidade!</div>`;
        divAnswer.innerHTML += `<div class="answer true">DIRIJA COM CUIDADO!</div>`;

    } else {

        divAnswer.innerHTML = `<div class="answer error">VOCÊ FOI MULTADO!</div>`;
        divAnswer.innerHTML += `<div class="answer error">O LIMITE DE VELOCIDADE É 80km/h!</div>`;
        divAnswer.innerHTML += `<div class="answer error">Por ter excedido ${inputKmConverted - 80}km acima do limite, Você deverá pagar R$${((inputKmConverted - 80) * 7).toFixed(2)}.</div>`;
        divAnswer.innerHTML += `<div class="answer error">DIRIJA COM CUIDADO!</div>`;
    }

    
}